chain/jsonrpc-primitives/src/types/network_info.rs

```diff
@@ -22,7 +22,7 @@ pub struct RpcKnownProducer {
 pub struct RpcNetworkInfoResponse {
     pub active_peers: Vec<RpcPeerInfo>,
     pub num_active_peers: usize,
-    pub peer_max_count: u32,
+    pub peer_max_count: usize,
```

Comment:
is this change backwards compatible? (I remember that json and u64 had issues when serializing - as you had to use strings in JSON).

Reply:
Yes, the change is backward compatible.

The only difference is that before, we wouldn't decode some specially prepared JSON messages. Instead, the process will fail on "validate()" method.

The main reason why you don't want to use `usize`, `u64` , is when you store values greater than `i64::MAX`, because those can't be represented by java. This matters when you store, for example 8 bytes hashes. However, that's not the case for us. So there is nothing to worry about.